src/core/Microsoft.PowerToys.Settings.Test/SettingsUtilsTests.cs

```diff
@@ -0,0 +1,99 @@
+using Microsoft.PowerToys.Settings.UI.Lib;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Microsoft.PowerToys.Settings.UnitTest
+{
+    [TestClass]
+    public class SettingsUtilsTests
+    {
+        [TestMethod]
+        public void SaveSettings_SaveSettingsToFile_WhenFilePathExists()
+        {
+            // Arrange
+            string file_name = "\\test";
+            string file_contents_correct_json_content = "{\"name\":\"powertoy module name\",\"version\":\"powertoy version\"}";
+
+            BasePTSettingsTest expected_json = JsonSerializer.Deserialize<BasePTSettingsTest>(file_contents_correct_json_content);
+
+            // Act
+            SettingsUtils.SaveSettings(file_contents_correct_json_content, file_name);
+            BasePTSettingsTest actual_json = SettingsUtils.GetSettings<BasePTSettingsTest>(file_name);
+
+            // Assert
+            Assert.IsTrue(actual_json.Equals(actual_json));
+        }
+
+        [TestMethod]
+        public async Task SaveSettings_ShouldCreateFile_WhenFilePathIsNotFoundAsync()
+        {
+            // Arrange
+            string file_name = "test\\Test Folder";
```

Comment:
I think we should clean the folder and files created after each test.

Reply:
https://github.com/microsoft/PowerToys/pull/2129/commits/1756d332fbe00938d366b0ea37147604660814bb